github-for-jira.sd.yml

```diff
@@ -225,16 +225,16 @@ environmentOverrides:
                       Value: { "Fn::GetAtt": [ "ALBTargetGroup", "TargetGroupFullName" ] }
                   MetricName: TargetResponseTime
                   Namespace: AWS/ApplicationELB
-                Period: 60
-                Stat: Average
+                Period: 300
+                Stat: Maximum
```

Comment:
The only thing which concerns me here is that Maximum might be a bit noisy here. However Micros doesn't allows percentiles and Average wouldn't work well (due to healtcheck, deepcheck). We can go with it anyway and then adjust if it create noise.